src/ubx-message.toit

```diff
@@ -2496,3 +2667,438 @@ class CfgGnss extends Message:
+    else:
+      if enable == null: put-uint8_ (4 + port) type
+      else if enable: put-uint8_ (4 + port) ((uint8_ (4 + port)) | type)
+      else: put-uint8_ (4 + port) ((uint8_ (4 + port)) & (~type & 0xFF))
+
+  /**
+  Gets the mask byte for the given $port.
+
+  $port must be one of $PORT-DDC(0), $PORT-UART1(1), $PORT-UART2(2),
+    $PORT-USB(3), $PORT-SPI(4), $PORT-RES5(5).
+  */
+  get-port-type-mask port/int -> int:
+    assert: payload.size >= 10
+    assert: 0 <= port <= 5
+    return uint8_ (4 + port)
+
+  /**
+  Sets the mask byte for outright the given $port.
+
+  $port must be one of  $PORT-ALL(-1), $PORT-DDC(0), $PORT-UART1(1),
+    $PORT-UART2(2), $PORT-USB(3), $PORT-SPI(4), $PORT-RES5(5).
+  */
+  set-port-type-mask port/int --mask/int -> none:
+    assert: payload.size >= 10
+    assert: -1 <= port <= 5
+    set-port-type --port=port --enable=null --type=mask
+
+  // Convenience per-port getters/setters (only valid for Get/Set form).
+  get-type-mask-ddc -> int: return get-port-type-mask PORT-DDC
+  get-type-mask-uart1 -> int: return get-port-type-mask PORT-UART1
+  get-type-mask-uart2 -> int: return get-port-type-mask PORT-UART2
+  get-type-mask-usb -> int: return get-port-type-mask PORT-USB
+  get-type-mask-spi -> int: return get-port-type-mask PORT-SPI
+  get-type-mask-res5 -> int: return get-port-type-mask PORT-RES5
+
+  set-type-mask-ddc mask/int -> none: set-port-type-mask PORT-DDC --mask=mask
+  set-type-mask-uart1 mask/int -> none: set-port-type-mask PORT-UART1 --mask=mask
+  set-type-mask-uart2 mask/int -> none: set-port-type-mask PORT-UART2 --mask=mask
+  set-type-mask-usb mask/int -> none: set-port-type-mask PORT-USB --mask=mask
+  set-type-mask-spi mask/int -> none: set-port-type-mask PORT-SPI --mask=mask
+  set-type-mask-res5 mask/int -> none: set-port-type-mask PORT-RES5 --mask=mask
+
+  // Helpers for checking whether a given INF type is enabled on a port.
+  error-enabled  port/int -> bool: return (get-port-type-mask port) & MASK-ERROR   != 0
+  warning-enabled port/int -> bool: return (get-port-type-mask port) & MASK-WARNING != 0
+  notice-enabled port/int -> bool: return (get-port-type-mask port) & MASK-NOTICE  != 0
+  test-enabled port/int -> bool: return (get-port-type-mask port) & MASK-TEST    != 0
+  debug-enabled port/int -> bool: return (get-port-type-mask port) & MASK-DEBUG   != 0
+
+  // Helpers for enabling a given INF type on a port.
+  enable-error port/int=-1 -> none: set-port-type --port=port --enable --type=MASK-ERROR
+  enable-warning port/int=-1 -> none: set-port-type --port=port --enable --type=MASK-WARNING
+  enable-notice port/int=-1 -> none: set-port-type --port=port --enable --type=MASK-NOTICE
+  enable-test port/int=-1 -> none: set-port-type --port=port --enable --type=MASK-TEST
+  enable-debug port/int=-1 -> none: set-port-type --port=port --enable --type=MASK-DEBUG
+  /**
+  Disable all message types for a specific port.
```

Comment:
Doesn't start with a 3rd person verb, and the function is "Enable" not "disable".

```suggestion
  Enables all message types for the given $port.
```